package/MDAnalysis/coordinates/XDR.py

```diff
@@ -220,10 +219,10 @@ def _load_offsets(self):
             # refer to Issue #1893
             data = read_numpy_offsets(fname)
             if not data:
-                warnings.warn("Reading offsets from {} failed, "
-                              "reading offsets from trajectory instead.\n"
-                              "Consider setting 'refresh_offsets=True' "
-                              "when loading your Universe.".format(fname))
+                warnings.warn(f"Reading offsets from {fname} failed, "
+                              f"reading offsets from trajectory instead.\n"
+                              f"Consider setting 'refresh_offsets=True' "
+                              f"when loading your Universe.")
```

Comment:
```suggestion
                              "reading offsets from trajectory instead.\n"
                              "Consider setting 'refresh_offsets=True' "
                              "when loading your Universe.")
```

Reply:
No need for f-strings here? (Unless there it's some black/formatting thing I'm not aware of...) IIRC some linters might complain.